### PR TITLE
Fix hcf references.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ref: https://trello.com/c/ZNBu0nTv/4-5-remove-remaining-references-to-hcf-hcp
:construction: Wait with merge, this has lots of related changes, see ticket.

### CODE REVIEW

File: docs/dev grade services/scripts/mongo [hunk 1]
```diff
@@ -21,9 +21,9 @@ docker run --name ${MONGO_CONTAINER_NAME} ${MONGO_DATA_CMD} -d ${MONGO_VERSION}
     CONTAINER_ID=`docker ps|grep -w ${MONGO_CONTAINER_NAME}|awk '{print $1}'`
     echo "container id: ${CONTAINER_ID}"
 
-    docker network connect hcf ${CONTAINER_ID}
+    docker network connect scf ${CONTAINER_ID}
```
Comment (mook-as) [inline docs/dev grade services/scripts/mongo:24]: I would be very surprised if this worked

**Review by mook-as — CHANGES_REQUESTED** [review]

File: docs/dev grade services/scripts/mongo (hunk 1 above)
Comment (andreas-kupries) [reply docs/dev grade services/scripts/mongo:24]: Yes, well, I do not know what I am doing here, and it is for something we do not use AFAIK, so I did not think too much on the change.
I suspect it needs setup from other (unknown to me) parts of the system.

File: make/include/versioning.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ GIT_TAG=${GIT_TAG:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $1 }' )}
 GIT_COMMITS=${GIT_COMMITS:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $2 }' )}
 GIT_SHA=${GIT_SHA:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $3 }' )}
 
-ARTIFACT_NAME=${ARTIFACT_NAME:-$(basename $(git config --get remote.origin.url) .git | sed s/^hcf-//)}
+ARTIFACT_NAME=${ARTIFACT_NAME:-$(basename $(git config --get remote.origin.url) .git | sed s/^scf-//)}
```
Comment (alexjh) [inline make/include/versioning.sh]: The sed doesn't have any effect here... I wonder if it ever did? I think we should just drop it.

**Review by alexjh — CHANGES_REQUESTED** [review]
I think the docs should be a separate PR for some other time, they need a lot of work.

File: make/include/versioning.sh (hunk 2 above)
Comment (andreas-kupries) [reply make/include/versioning.sh]: Addressed with commit debf4cf3e16229e550e3564ae03dab530725ec0b

**Pull request merged**